### PR TITLE
User Pages w/ API Calls

### PR DESCRIPTION
**Pull request by hashmaster3k**
Creates view pages for:

- User profile via ```/profile``` or ```profile_path```
- User edit page via ```/profile/:id/edit``` or ```edit_user_path```

The user has the ability to delete their profile as well with a button on their profile page. 

All calls are web-mocked. Using environment variable BE_URL for the base url to the back end app on heroku.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -36,7 +36,7 @@
         </ul>
         <ul class="nav justify-content-end">
           <li class="nav-item">
-            <a class="nav-link" href="<%=dashboard_path%>">Profile</a>
+            <a class="nav-link" href="<%=profile_path%>">Profile</a>
```
Comment (hashmaster3k) [inline app/views/layouts/application.html.erb:39]: Altered Nav-bar links to make more sense and work.

Comment (LHJE) [reply app/views/layouts/application.html.erb:39]: Yeah that makes way more sense

File: spec/facades/user_facade_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  it 'delete_user() returns nothing' do
+    stub_request(:delete, "#{ENV['BE_URL']}/api/v1/users/#{@user.id}").
+       to_return(status: 200, body: "", headers: {})
+    results = UserFacade.delete_user(@user.id)
+    expect(results).to be_a(Faraday::Response)
+    expect(results.body).to be_a(String)
+    expect(results.body).to be_empty
+  end
+end
```
Comment (LHJE) [inline spec/facades/user_facade_spec.rb:37]: Very cool, great job!

File: spec/features/navbar/navbar_spec.rb [hunk 3]
```diff
@@ -59,7 +59,7 @@
     end
 
     it "can see my gardens, my impact, learn more, profile and logout on profile page" do
-      visit "/users/#{@user.id}"
+      visit profile_path
 
       within "#navbar-#{@user.id}" do
         expect(page).to have_link('My Gardens')
```
Comment (LHJE) [inline spec/features/navbar/navbar_spec.rb:65]: So restful!

**Review by LHJE — APPROVED** [review]
Looks terrific.  Thanks Hash!

**Pull request merged**